hrm-service/src/contact-job/contact-job-processor.ts

```diff
@@ -0,0 +1,40 @@
+import { setInterval } from 'timers';
+import { processCompleteContactJobs } from './contact-job-complete';
+import { publishPendingContactJobs } from './contact-job-publish';
+
+let processingJobs = false;
+
+const JOB_PROCESSING_INTERVAL_MILLISECONDS = 5000;
+const JOB_RETRY_INTERVAL_MILLISECONDS = 60000;
+
+export const getProcessingInterval = () => JOB_PROCESSING_INTERVAL_MILLISECONDS;
```

Comment:
Rather than exposing this (which I assume is purely for testing), it might be more flexible (and produce faster, more deterministic test code) if you exposed an awaitable function used to delay the processing / retry loop. 

This way you can mock it with something that returns a promise success straight away if you don't care about waiting, or you can mock it with a promise that you resolve at a specific point in the flow of your test if you want to test scenarios where things change in between processing 'sweeps' .

Or you could just mock 'setTimeout' directly in your tests and not expose anything here for testing, either would work